### DISCUSSION TITLE
Where to get EM272 or AOM-5024 at sensible price in UK?

### DISCUSSION THREAD

**Discussion by SheddyIan** [mcguirepr89/BirdNET-Pi#487] [Q&A] [answered]
I'm very new to all this, I set up BirdNET-Pi 3 days ago on a Raspberry Pi 3B+.  I'm using a cheap Chinese USB sound card that I've had for years and seems reliable, and tried a couple of cheap microphones on it.

One - a computer desktop microphone branded "Trust" seemed the most sensitive, but had an appalling hum.  I took it apart (well, broke it apart) and discovered that the audio lead was two incredibly thin wires and no screen!

Changed the cable for a screened one and the hum is gone.  It all works quite well.  I've been pleased at the birds I've detected, and find it fascinating.

BUT

It often doesn't pick up birds that I can hear tweeting from indoors through a closed window.  (The microphone is outside, quite close to the window I can hear the birds through)

Reading the helpful article I see that the EM272 or AOM-5024 are suggested modules.

But the price in the UK seems huge - Mouser wanting £20 delivery for one tiny module!

Does anyone know of a sensibly priced place that I might order one from?

Or failing that, a suggestion for an alternative good microphone or module that will do well.

I'm on a tight budget so am hoping to keep it cheap

thanks

Ian

**Comment by ehpersonal38** [comment, marked as answer]
You checked this site out?

https://micbooster.com/clippy-and-pluggy-microphones/65-clippy-em172-microphone.html?search_query=EM272+&results=28
https://micbooster.com/microphone-capsules/199-primo-em272.html?search_query=Primo+EM272&results=56

**Reply by Svardsten53** [reply]
Or this one?
https://www.reichelt.com/en/en/electret-capacitor-microphone-capsule-aom-5024l-hd-r-p285350.html?r=1

**Reply by SheddyIan** [reply]
That's a good price, but they don't seem to ship to UK.. I can see every other country there.  Either I'm overlooking it in the list or it's another Brexit bonus!

thanks for the suggestion tho

**Reply by SheddyIan** [reply]
> You checked this site out?
> 
> https://micbooster.com/clippy-and-pluggy-microphones/65-clippy-em172-microphone.html?search_query=EM272+&results=28 https://micbooster.com/microphone-capsules/199-primo-em272.html?search_query=Primo+EM272&results=56

Thank you, the 2nd one comes to a total of £15.48 with VAT and delivery, but that's the best price so far.

I'll leave this question open a little longer in case someone else knows of a cheaper one, otherwise I'll mark this as the answer.

thanks!

**Comment by ehpersonal38** [comment]
On an unrelated note I feel like there's a business opportunity to be had soldering up these capsules into a ready-to-use microphone and selling them to people like us. 

Hell why don't the cheap microphone manufacturers on Amazon do this? Why do they use such terrible hardware when these capsules are so inexpensive?

**Reply by SheddyIan** [reply]
> On an unrelated note I feel like there's a business opportunity to be had soldering up these capsules into a ready-to-use microphone and selling them to people like us.
> 
> Hell why don't the cheap microphone manufacturers on Amazon do this? Why do they use such terrible hardware when these capsules are so inexpensive?

A variant of this occurred to me earlier when looking at the prices (deliver, mostly) that I, or someone more enterprising, could buy a lot up and re-sell them on ebay, so the individual item cost would be lower.

Your idea is also good - a simple pre-made AOM-5024 soldered to a reasonable quality grounded lead and terminated in a 3.5mm plug

Is there really the market for this tho?

**Reply by ehpersonal38** [reply]
> Is there really the market for this tho?

I don't know the exact numbers but I think a few dozen people a day install BirdNET-Pi onto their systems, maybe 1/3rd don't have a mic and go to Amazon or an equivalent place to buy a cheap one. Say 10% will actually buy from you instead... so around 1 sale a day? Hehe

**Reply by adamhp** [reply]
Here I am 3 years later Googling this exact thing ;)
